qiita_db/metadata_template/test/test_prep_template.py

```diff
@@ -32,7 +32,8 @@
                            get_count)
 from qiita_db.metadata_template.prep_template import PrepTemplate, PrepSample
 from qiita_db.metadata_template.sample_template import SampleTemplate, Sample
-from qiita_db.metadata_template.constants import PREP_TEMPLATE_COLUMNS
+from qiita_db.metadata_template.constants import (
```

Comment:
these values are in the __init__.py file, so you can import directly from `qiita_db.metadata_template`